doc/classes/Timer.xml

```diff
@@ -15,15 +15,16 @@
 		<method name="is_stopped" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the timer is stopped.
+				Returns [code]true[/code] if the timer is stopped or has not started.
 			</description>
 		</method>
 		<method name="start">
 			<return type="void" />
 			<param index="0" name="time_sec" type="float" default="-1" />
 			<description>
-				Starts the timer. Sets [member wait_time] to [param time_sec] if [code]time_sec &gt; 0[/code]. This also resets the remaining time to [member wait_time].
-				[b]Note:[/b] This method will not resume a paused timer. See [member paused].
+				Starts the timer, if it was not started already. Fails if the timer is not inside the tree.
+				If [param time_sec] is greater than [code]0[/code], this value is used for the [member wait_time].
+				[b]Note:[/b] This method does not resume a paused timer. See [member paused].
```

Comment:
I suggest reducing the number of paragraphs to make the description more compact:

```suggestion
				Starts the timer, if it was not started already. Fails if the timer is not inside the tree. If [param time_sec] is greater than [code]0[/code], this value is used for the [member wait_time].
				[b]Note:[/b] This method does not resume a paused timer. See [member paused].
```